core/util/runChromy.js

```diff
@@ -9,114 +9,16 @@ const CHROMY_STARTING_PORT_NUMBER = 9222;
 const DEFAULT_FILENAME_TEMPLATE = '{configId}_{scenarioLabel}_{selectorIndex}_{selectorLabel}_{viewportIndex}_{viewportLabel}';
 const DEFAULT_BITMAPS_TEST_DIR = 'bitmaps_test';
 const DEFAULT_BITMAPS_REFERENCE_DIR = 'bitmaps_reference';
-// const BACKSTOP_TOOLS_PATH = '/capture/backstopTools.js';
 const SELECTOR_NOT_FOUND_PATH = '/capture/resources/selectorNotFound_noun_164558_cc.png';
 const HIDDEN_SELECTOR_PATH = '/capture/resources/hiddenSelector_noun_63405.png';
 const BODY_SELECTOR = 'body';
 const DOCUMENT_SELECTOR = 'document';
 const NOCLIP_SELECTOR = 'body:noclip';
 const VIEWPORT_SELECTOR = 'viewport';
 
+const injectBackstopTools = require('../../capture/backstopTools.js');
 const BackstopException = require('../util/BackstopException.js');
 
-// ============================== MOVE TO NEW FILE VVVVV ===========================
```

Comment:
Please move these functions into the new file include verbatim -- these have been tested.  Let's not use the original backstopTools methods.  Thanks.

Reply:
I'm not sure i understand your request. You want those methods from `runChromy.js` to be moved to `backstopTools.js` as is? Isn't that the case? I saw no difference aside from the usage of constants in for the original methods in `backstopTools.js`.

Reply:
Thanks for looking at this.  Yes, That is the request.   I think there may have been tweaks to the methods inside of `runChromy.js`  -- That code has been working in prod -- so I'd like to ensure there are as few changes as possible.    I think there may be some minimal inline documentation added as well.   Simply copying the whole thing may be easier than attempting to diff.

Reply:
We may also want to extend the names to avoid any collisions.  I never liked the name .exists -- it's way too common.  maybe `window._backstopTools.exists()` would be better.